app/(home)/visitreport.tsx

```diff
@@ -1,12 +1,330 @@
+import { Formik, FormikHelpers, FormikValues } from "formik";
+import { ClientVisitFormInitialData } from "@/apptypes/AppStaticData";
+import { ClientVisitFormValidationSchema } from "@/apptypes/AppValidationSchemas";
+import StyledTextAreaInput from "@/components/formcontrols/styledTextAreaInput";
+import { router } from "expo-router";
+import { getLocationCoordinates } from "@/lib/appwrite";
+import * as AppType from "@/apptypes/AppTypes";
+import moment from "moment";
+import LoadingControl from "@/components/loading";
+import { postMethod } from "@/lib/appAPIServices";
+
+const validationSchema = ClientVisitFormValidationSchema;
+
+const VisitReport = () => {
+  const [initialFormData, setInitialFormData] =
+    useState<AppType.ClientVisitFormData>(ClientVisitFormInitialData);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fectLocation = async () => {
+      let dateVisit = moment(new Date()).format("DD/MM/YY");
+      let timeVisit = moment(new Date().getTime()).format("HH:mm");
+      const locationVal = await getLocationCoordinates();
+      if (locationVal != undefined && locationVal != null) {
+        const setInitData = {
+          ...initialFormData,
+          dateOfVisit: dateVisit,
+          timeOfVisit: timeVisit,
+          latCode: locationVal.locationCode.latitude.toString(),
+          longCode: locationVal.locationCode.longitude.toString(),
+          address1: locationVal.address.address1,
+          address2: locationVal.address.address2,
+          state: locationVal.address.state,
+          city: locationVal.address.city,
+          pincode: locationVal.address.pincode,
+        };
+        setInitialFormData(setInitData);
+      }
+    };
+    fectLocation();
+  }, []);
+
+  const submitClientForm = async (values: FormikValues, action: FormikHelpers<AppType.ClientVisitFormData>) => {
+    try {
+      setIsLoading(true);
+      const userId = "55655";
+      const reqBody: AppType.ClientVisitRequest = {
+        cvdSeqid: "0",
+        cvdName: values.name,
+        cvdMobile: values.mobileNo,
+        cvdProduct: values.interestPrd,
+        cvdloanAmtReq: values.loanAmt,
+        cvdLatitude: values.latCode,
+        cvdLongitude: values.lonCode,
+        cvdAddress1: values.address1,
+        cvdAddress2: values.address2,
+        cvdState: values.state,
+        cvdCity: values.city,
+        cvdPincode: values.pincode,
+        cvdRemarks: values.remarks,
+        cvdLocation: "", // branch
+        cvdLeadType: values.leadCategory,
+        cvdVisitDate: values.dateOfVisit,
+        cvdVisitTime: values.timeOfVisit,
+        cvdCreatedBy: "55655",
+        cvdCreatedOn: "",
+        cvdModifiedBy: "55655", // username
+        cvdModifiedOn: "",
+      };
+      let resp = await postMethod(
+        AppType.APIClassName.leadManegment,
+        `${AppType.APIMethods.saveClientVisit}/${userId}`,
+        reqBody
+      );
+      console.log("fjkdjs", resp);
```

Comment:
use console.info , never use console.log

Reply:
sure. next time i will not repeat